### PR TITLE
Assigning skills and creating questions

### PR DESCRIPTION
**Pull request by audstav**
@seanlip @DubeySandeep PTAL

Updates on:
https://oppia-user-guide.readthedocs.io/en/latest/index.html

Forgot to add: 'Guide to lesson creation' section has been updated too to incorporate additional lesson creation guidelines.

### CODE REVIEW

**Comment by seanlip** [comment]
@aks681 Could you PTAL at this and leave feedback? Thanks!

File: docs/admins/assignskill.rst [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+   *Fig 1.*
+
+1. To assign a skill to a topic, click the **Assign to Topic** button next to the skill.
+2. Select the Topic to which you want to assign the skill and click |savebutton|. As you assign skills, they are taken out of the **Unused Skills** tab and added to the topic.
+
+.. |savebutton| image:: /images/save_button.png
+                :scale: 40 %
+
+3. To merge a an unused skill with a skill already assigned to a topic, click the **Merge Skill** button next to the skill, and select the skill you want to merge with.
```
Comment (aks681) [inline docs/admins/assignskill.rst]: Remove the extra 'a'

File: docs/admins/assignskill.rst [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+Questions can be created either through the Topic Editor or submitted by contributors. 
+
+1. To create a question from the Topic Editor, click on the topic, and then the **Questions Editor** |questioneditor| button in the top-right corner.
+
+.. |questioneditor| image:: /images/question_editor.png
+                    :scale: 30 %
+
+2. Click the **Create Question** button.
+
+3. Select the skill for which you want to create the question and click **Next**.
```
Comment (aks681) [inline docs/admins/assignskill.rst]: Here, you can select multiple skills as well, maybe mention that as well?

File: docs/admins/assignskill.rst [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+.. figure:: /images/select_skill_question.png
+   :alt: Selecting skills for questions
+   :scale: 30 %
+
+   *Fig. 6*
+
+4. Select the question's level of difficulty—Easy, Medium or Hard—and click **Done**. If there were :ref:`rubrics <rubrics>` written for these levels when the skill was created, you will see them here.
+
+5. The **Question Editor** box opens. This works just like an :ref:`Exploration Editor <createcontent>` where you can create questions, add interactions, responses and feedback. You also have to add a hint in order to publish the question. When you have finished creating the question, click |publishquestion|.
```
Comment (aks681) [inline docs/admins/assignskill.rst]: Here, for interactions that have solutions (eg: TextInput has it, but multiple choice does not), that also has to be specified.

**Review by aks681 — COMMENTED** [review]
Just minor comments, otherwise this looks good!
Also, saw the video and looks nice!

File: docs/admins/assignskill.rst (hunk 1 above)
Comment (audstav) [reply docs/admins/assignskill.rst]: Oops - done!

File: docs/admins/assignskill.rst (hunk 2 above)
Comment (audstav) [reply docs/admins/assignskill.rst]: Done

File: docs/admins/assignskill.rst (hunk 3 above)
Comment (audstav) [reply docs/admins/assignskill.rst]: Done

**Comment by audstav** [comment]
Incorporated changes and new commit added, PTAL

**Review by aks681 — APPROVED** [review]
LGTM!

**Review by seanlip — APPROVED** [review]
Looks good, thanks @audstav! Merging.

**Pull request merged**